.travis.yml

```diff
@@ -106,6 +106,12 @@ matrix:
               false;
           fi
 
+    - <<: *basic-vm
+      name: "UTF-8 Check"
+      script:
+        # Make sure we're not introducing any non-UTF-8 character in text files
```

Comment:
Do you mean "non-ASCII" character?

Reply:
A possible utf-8 check is
```
file -I <filename>
```
and the output should contain `charset=utf-8`.

Reply:
Ah see above 😄 ⬆️